components/renku_data_services/project/blueprints.py

```diff
@@ -33,7 +40,20 @@ def get_all(self) -> BlueprintFactoryResponse:
         async def _get_all(_: Request, *, user: base_models.APIUser, pagination: PaginationRequest):
             projects, total_num = await self.project_repo.get_projects(user=user, pagination=pagination)
             return [
-                apispec.Project.model_validate(p).model_dump(exclude_none=True, mode="json") for p in projects
+                dict(
```

Comment:
So I tried to use this 
```
            return json(
                apispec.Project.model_construct(**asdict(result), etag=result.etag).model_dump(
                    exclude_none=True, mode="json"
                ),
                201,
            )
```

But I keep getting weird failures with `AttributeError: 'str' object has no attribute 'root'` from `model_dump`. And I tried a bunch of different things with no success. I also had to manually add `etag` because it is a property so it does not get included in the output of `asdict`. So I think we should leave it as is for now.